template.php

```diff
@@ -150,6 +150,29 @@ function bootstrap5_lite_preprocess_page(&$variables) {
       }
     }
   }
+
+  // Add 'not-front' if we're not front.
+  if (!$variables['is_front']) {
+    $variables['classes'][] = 'not-front';
+  }
+
+  // Add classes based on normal path parts.
+  $normal_path = strtolower(backdrop_get_normal_path(request_path()));
+  $normal_path = str_replace('_', '-', $normal_path);
+  $path_parts = $normal_path ? explode('/', $normal_path) : array();
```

Comment:
Wouldnt `$variables['classes'][] = 'page-' . str_replace('/', '-', $normal_path)` accomplish the same thing here as the code following?

Reply:
The code below would take the path `foo/bar/baz` and add the classes `page-foo`, `page-foo-bar`, and `page-foo-bar-baz`, like the Drupal Bootstrap theme does.